frontends/infinite-corridor/src/pages/SearchPageAlt.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { assertInstanceOf } from "ol-util"
+import { screen, renderTestApp, setMockResponse } from "../test-utils"
+import { fireEvent, waitFor } from "@testing-library/react"
+import { makeSearchResponse } from "../factories/search"
+import { makeRequest } from "../test-utils/mockAxios"
+
+/**
+ * This is a bit of a hack to enable `react-infinite-scroller` to respond to
+ * scroll events in jsdom. This will be somewhat fragile as it relies on the
+ * current implementation of scrolling in `react-infinite-scroller`.
+ *
+ * - `react-infinite-scroller` does not respond to scroll events on an element
+ *    if the element does not have an `[offset parent](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent)`
+ * - JSDOM does not implement `offsetParent` property (it is always null).
+ *
+ * So this patches `offsetParent` to return `document.body`
+ */
+const enableInfiniteScrollerScrolling = async () => {
+  const results = await screen.findByLabelText("Search Results", {
+    exact: false
+  })
+  // eslint-disable-next-line testing-library/no-node-access
+  const scroller = results.parentNode
+  assertInstanceOf(scroller, HTMLElement)
+  jest.spyOn(scroller, "offsetParent", "get").mockReturnValue(document.body)
+}
```

Comment:
Context: Enzyme gives devs access to the react component itself. So you can do stuff like 

```ts
// call `loadMore` ourselves
wrapper.get(InfiniteScroller).prop('loadMore')()
```

react-testing-library doesn't give you direct access to the components (neither does react, at least via its public api; enzyme gives you access to the components vai react internals). RTL's philosophy is usually: do the "real thing" (click a button; type text, etc) and check that your component responds appropriately. That's particularly hard, at least in JSDOM, with scrolling.